accounting/service.proto

```diff
@@ -1,17 +1,17 @@
 syntax = "proto3";
 
-package accounting;
+package neo.fs.accounting.v2;
```

Comment:
Don't you want to use `neo.fs.v2.package-name`? This will match `neofs-api-go` import structure.

Reply:
typo. fixed.